olm/openshift-pipelines-operator.resources.yaml

```diff
@@ -96,7 +87,7 @@ data:
           capabilities: Basic Install
           categories: "Developer Tools, Integration & Delivery"
           description: OpenShift Pipelines is a cloud-native CI/CD solution for building pipelines using Tekton concepts which run natively on OpenShift and Kubernetes.
-          containerImage: quay.io/openshift-pipeline/openshift-pipelines-operator:v0.3.1-2
+          containerImage: REPLACE_IMAGE
```

Comment:
@nikhil-thomas should this be reverted?

Reply:
Yes,
`olm/openshift-pipelines-operator.resources.yaml` is a generated file. so build an image, udpate image in CSV (0.5.0) then run `scripts/olm_catalog.sh`